src/codegen/object.rs

```diff
@@ -440,7 +440,6 @@ impl {name}Builder {{
         "
     // rustdoc-stripper-ignore-next
     /// Build the [`{name}`].
-    #[must_use = \"The builder must be built to be used\"]
     pub fn build(self) -> {name} {{
```

Comment:
This is actually semi-useful: if you don't do anything with the return value of `build()` then you're probably doing something wrong.